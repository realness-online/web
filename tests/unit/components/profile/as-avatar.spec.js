import { shallow } from 'vue-test-utils'
import as_avatar from '@/components/profile/as-avatar'
const person = {
  first_name: 'Scott',
  last_name: 'Fryxell',
  id: '/+14151234356'
}
describe('@/components/profile/as-avatar.vue', () => {
  let wrapper
  beforeEach(() => {
    wrapper = shallow(as_avatar, {
      propsData: { person }
    })
  })
  it('Render an avatar', () => {
    expect(wrapper.element).toMatchSnapshot()
  })
})
