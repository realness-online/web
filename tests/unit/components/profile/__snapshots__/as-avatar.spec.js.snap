// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`@/components/profile/as-avatar.vue Render an avatar 1`] = `
<svg>
  <defs
    itemprop="avatar"
  />
   
  <use
    xlink:href="#silhouette"
  />
</svg>
`;
